app-office/lyx/lyx-2.3.4.2.ebuild

```diff
@@ -0,0 +1,174 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{5,6,7,8} )
+
+MY_P="${P/_}"
+inherit desktop flag-o-matic font python-single-r1 qmake-utils toolchain-funcs xdg-utils
+
+DESCRIPTION="WYSIWYM frontend for LaTeX, DocBook, etc."
+HOMEPAGE="https://www.lyx.org/"
+SRC_URI="ftp://ftp.lyx.org/pub/lyx/stable/2.3.x/${MY_P}.tar.xz
+	ftp://ftp.lyx.org/pub/lyx/devel/lyx-2.3/${MY_P}/${MY_P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 ~hppa ~ppc ~ppc64 ~sparc x86 ~x64-macos ~x86-macos"
```

Comment:
can't add a version bump with stable keywords, move all to unstable (`ekeyword ~all lyx-2.3.4.2.ebuild`).

Reply:
done